static/css/drawer.css

```diff
@@ -1,132 +1,108 @@
-  font-size: 18px;
-  font-weight: bolder;
-  background-color: #ccc;
-  padding: 5px 10px;
-  border-radius: 5px;
-  border: 2px solid #aaa;
-  box-shadow: 2px 2px 0 #eee;
-  width: 100%;
+    font-size: 18px;
+    font-weight: bolder;
+    background-color: #ccc;
+    padding: 5px 10px;
+    border-radius: 5px;
+    border: 2px solid #aaa;
+    box-shadow: 2px 2px 0 #eee;
+    width: 100%;
 }
 
 #drawer {
-  background-color: #666;
-  border-left: 1px solid #999;
-  box-shadow: inset 4px 4px 5px 0 rgba(50, 50, 50, 0.75);
-  color: #eee;
-  display: none;
-  font-size: 16px;
-  height: 100%;
-  overflow-y: auto;
-  position: absolute;
-  margin-top: 45px;
-  right: -200px;
-  width: 300px;
-  top: 0;
-  z-index: 1;
+    background-color: #666;
+    border-left: 1px solid #999;
+    box-shadow: inset 4px 4px 5px 0px rgba(50, 50, 50, 0.75);
+    color: #eee;
+    display: none;
+    font-size: 16px;
+    height: 100%;
+    overflow-y: auto;
+    position: absolute;
+    margin-top: 45px;
+    right: -200px;
+    width: 300px;
+    top: 0;
+    z-index: 1;
 }
 
 #drawer i, #treatmentDrawer i {
-  opacity: 0.6;
+    opacity: 0.6;
 }
 
 #drawer a, #treatmentDrawer a {
-  color: white;
+    color: white;
 }
 
 input[type=number]:invalid {
-  background-color: #FFCCCC;
+    background-color: #FFCCCC;
 }
 
 #drawer .actions {
-  margin: 10px 2px;
+    margin: 10px 2px;
 }
 
 #drawer .actions a {
-  display: block;
-  text-align: right;
-  padding: 10px 0;
+    float: right;
+    padding: 10px 0;
 }
 
 #treatmentDrawer {
-  background-color: #666;
-  border-left: 1px solid #999;
-  box-shadow: inset 4px 4px 5px 0 rgba(50, 50, 50, 0.75);
-  color: #eee;
-  display: none;
-  font-size: 16px;
-  height: 100%;
-  overflow-y: auto;
-  position: absolute;
-  margin-top: 45px;
-  right: -200px;
-  width: 300px;
-  top: 0;
-  z-index: 1;
+    background-color: #666;
+    border-left: 1px solid #999;
+    box-shadow: inset 4px 4px 5px 0px rgba(50, 50, 50, 0.75);
+    color: #eee;
+    display: none;
+    font-size: 16px;
+    height: 100%;
+    overflow-y: auto;
+    position: absolute;
+    margin-top: 45px;
+    right: -200px;
+    width: 300px;
+    top: 0;
+    z-index: 1;
 }
 
 #treatmentDrawer input {
   box-sizing: border-box;
 }
 
-#treatmentDrawer .left-column {
+#treatmentDrawer label.left-column {
   display: block;
   width: 100%;
 }
 
-#treatmentDrawer .left-column span {
+#treatmentDrawer label.left-column span {
   width: 110px;
   display: inline-block;
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
-  line-height: 22px;
-  vertical-align: middle;
 }
 
-#treatmentDrawer .left-column input, #treatmentDrawer .left-column select {
+#treatmentDrawer label.left-column input, #treatmentDrawer label.left-column select {
   width: 140px;
 }
 
-#treatmentDrawer .extra-space {
+#treatmentDrawer label.extra-space {
   padding-top: 10px;
   display: block;
 }
 
-.timeago-mins {
-  width: 25px;
+input.timeago-mins {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Element (input.timeago-mins) is overqualified, just use .timeago-mins without element name.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/969293252/issues/source?bid=2452374&fileBranchId=2861014#l90)